mitosheet/src/plugin.tsx

```diff
@@ -337,6 +337,11 @@ function activateMitosheetExtension(
             // First, get the mito container that this element is a part of
             const mitoContainer = getParentMitoContainer();
 
+            // If we are in an input or text, we don't actually do the undo, as it's handled in the input
+            if (document.activeElement?.tagName.toLowerCase() === 'input' || document.activeElement?.tagName.toLowerCase() === 'textarea') {
+                return;
+            }
+
```

Comment:
Is this supposed to let me use `CMD + Z` to undo in the text area? When I use `CMD + Z` in the text area, the Mito Undo is not happening, but also the undo in the text area is not happening.

Reply:
This just makes it so the undo + redo. Jupyter lab integration issues prevent us from doing more.